server/chat-commands/info.ts

```diff
@@ -2530,6 +2530,72 @@ export const commands: ChatCommands = {
 	},
 	denyshowhelp: [`/denyshow [user] - Denies the media display request of [user]. Requires: % @ # &`],
 
+	randquote(target, room, user) {
+		room = this.requireRoom();
+		if (!room.settings.quotes?.length) return this.errorReply(`This room has no quotes.`);
+		this.runBroadcast();
+		const {quote, date, userid} = room.settings.quotes[Math.floor(Math.random() * room.settings.quotes.length)];
+		const formatted = quote.split('\n').map(item => Chat.formatText(item)).join('<br />');
+		return this.sendReplyBox(
+			`${formatted}<br />` +
+			`<hr /><small>Added by ${userid} on ${Chat.toTimestamp(new Date(date), {human: true})}</small>`
```

Comment:
Do most quote databases show this data?

Reply:
No, but i though that's what you meant by attribution? 
Since if you're not showing it here, there's little point, considering modlog exists.

Reply:
I don't think it needs to show author and date by default.

Reply:
Okay.

Reply:
Updated. It will now only show who it's by / when if you do !randquote showauthor.

Reply:
Hm... You could make an argument for it either way, I guess. :s